uscope/app/argus/widgets.py

```diff
@@ -1005,14 +1008,26 @@ def getProgressLayout():
             return layout
 
         layout = QVBoxLayout()
+        snap_layout = QHBoxLayout()
 
         layout.addLayout(getNameLayout())
 
         self.snapshot_pb = QPushButton("Snap")
         self.snapshot_pb.setIcon(QIcon(config.GUI.icon_files["camera"]))
         self.snapshot_pb.clicked.connect(self.take_snapshot)
-        layout.addWidget(self.snapshot_pb)
+        snap_layout.addWidget(self.snapshot_pb)
 
+        add_scalebar_label = QLabel("Add scalebar to image")
```

Comment:
Long term we might want to rethink this to keep UI simple (menu option?) but this work for MVP

Reply:
I can change the widget type, I put this checkbox according to the documentation